### DISCUSSION TITLE
Update 2022-08-08

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#5] [Announcements]
Greetings,

I have been refactoring the code into kernel and plug-in directories, and working on the demos.

We are looking at the option of porting the code to sbcl with a web-based CLOG interface.

If that succeeds and we have a cross-platform solution, I hope to make an initial developer preview release.

Some videos and documentation will follow.

Kaveh

**Comment by awolven** [comment]
What's your email address?

**Comment by kaiwulf** [comment]
I'm excited to help with this project!

**Reply by kaveh808** [reply]
Cool. Could you tell me a bit about your interests and experience?

**Reply by kaveh808** [reply]
Please reply in the Introductions message.

**Comment by kaiwulf** [comment]
What's the status on porting it to sbcl? I only use Linux and I use sbcl. I can't afford to get an Apple at this time. I'd happily contribute towards porting it towards sbcl.

**Comment by kaveh808** [comment]
I just pushed a change to make porting simpler.

In kons.asdf, make sure minimal-ui is included and ui is commented out.

All the device/graphics code should be in minimal-ui.lisp and opengl.lisp

Good luck! :)

**Comment by kaveh808** [comment]
Emailed you.
